constitution.tex

```diff
@@ -770,6 +770,9 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Overriding Directorships Qualifications}
+Qualifications pertaining to a specific directorship election can be overridden via a Constitutional Override, as explained in \ref{Semantic Changes}. Overrides will apply to a specific qualification of the directorship, and if passed, will be overridden for all candidates for said directorship.
```

Comment:
nit: this reads like it's permitting overrides, not clarifying the process.

Reply:
`When executing a Constitutional Override \ref{Semantic Changes} for a Qualification specified in \ref{Qualifications}, as explained in  the Override shall apply to a specific qualification of the directorship, and if passed, to all candidates for said directorship.`

Opinion on this rewrite?

Reply:
While I'm thinking of it, I'd also ask that you change the title of this subsection, since `Directorship` is poorly defined. Perhaps something like this, or maybe `Process` or `Clarifications` etc.
```tex
\asubsection{Guidelines on Overriding Executive Board Qualifications}
This section provides no definitions, instead serving to clarify the applicability of Overrides to Qualifications in the Selection of members of the Executive Board.
```


Might be good to refer to the specific sections that this refers to.
```tex
The Qualifications defined in \ref{Qualifications to be the Chairperson, Evaluations Director}, \ref{Qualifications to be the Social Director(s), Financial Director, Research and Development Director(s), House Improvements Director, House History Director}, \ref{Qualifications to be the Operational Communications Director}, and \ref{Qualifications to be the House Secretary or an Ad Hoc Director} apply to the entire Selection process.
```
That's not very nice to read though. Is the [cleveref package](https://en.wikibooks.org/wiki/LaTeX/Labels_and_Cross-referencing#The_cleveref_package) included with texlive and available on our systems? If so, that could be reduced to 
```tex
The qualifications defined in \crefrange{Qualifications to be the Chairperson, Evaluations Director}{Qualifications to be the House Secretary or an Ad Hoc Director} apply to the entire Selection process.
```

This is my first attempt at being a bit stricter in wording. Is this clear? Maybe we can/should work in an illustrative example.
```tex
As a result, when these sections are Overridden, the Override must apply similarly to the entirety of the Selection Process.

As these Qualifications are defined for the entire Selection, an Override is not capable of overriding the Qualifications for only specific candidates in the Selection.
```